dummy/controllers/menuController/menuController.js

```diff
@@ -0,0 +1,45 @@
+import menuDb from '../../db/menu';
+import fieldValidation from '../../validation/mealFieldRequiredValidation';
+
+// const { mealFieldRequiredValidation } = Validation;
+
+// eslint-disable-next-line require-jsdoc
+export default class Menu {
+  // eslint-disable-next-line valid-jsdoc
+  /**
+   *
+   * @param {*} req  THE REQUEST OBJECT
+   * @param {*} res   THE RESPONSE OBJECT
+   * @param {*} json  THE  RETURN JSON
+   */
+  static setMenu(req, res) {
+    const {
+      img, title, description, price
+    } = req.body;
+    fieldValidation(img, title, description, price, res);
+
+    const result = menuDb.filter(
+      (Title) => Title.title === title.toLowerCase()
+    );
+
+    if (!result.length < 1) {
+      return res.status(400).json({ message: 'Menu  already set' });
```

Comment:
The correct status code to use here is `409`, not `400`. This is because a meal with the title already exists. 400 is used for general unresolved client errors. Check out this [link](https://httpstatuses.com/) to learn more about status codes.